javascript/253-Meeting-Rooms-ii.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @param {number[][]} intervals
+ * Time O((N * logN) + (M * logM)) | Space O(1)
+ * @return {number}
+ */
+ var minMeetingRooms = function(intervals) {
+    const { start, end } = splitIntervals(intervals);
+    let [ minRooms, startIndex, endIndex ] = [ 0, 0, 0 ];
+
+    while (startIndex < intervals.length) {
+        const [ startTime, endTime ]  = [ start[startIndex], end[endIndex] ];
+
+        const hasGap = (prevEnd, currStart) =>
```

Comment:
nit: prefer boolean declaration instead of the trivial lambda:
`const hasGap = endTime < startTime;`